src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -646,6 +648,23 @@ Namespace Microsoft.CodeAnalysis.Operations
             Return New InvalidOperation(children, _semanticModel, syntax, type, constantValue, isImplicit)
         End Function
 
+        Private Function CreateBoundTypeArgumentsOperation(boundTypeArguments As BoundTypeArguments) As IInvalidOperation
+            ' This can occur in scenarios involving unresolved extension methods in Strict mode, such as
```

Comment:
>This can occur in scenarios involving unresolved extension methods in Strict mode [](start = 14, length = 81)

If the method is no where to be found, why are we assuming it is an extension method?